site/cmdpalette.js

```
@@ -0,0 +1,516 @@
+      case 'ArrowUp':
+        e.preventDefault();
+        if (!count) return;
+        _activeIdx = (_activeIdx - 1 + count) % count;
+        _updateActive(items);
+        break;
+
+      case 'Enter':
+        e.preventDefault();
+        var target = (_activeIdx >= 0 && items[_activeIdx])
+          ? items[_activeIdx]
+          : (count === 1 ? items[0] : null);
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)